test/jobs/benchmark_regression_job_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
+
+class BenchmarkRegressionJobTest < ActiveJob::TestCase
+  test "#perform benchmark_regression" do
+    benchmark_result_type = create(:benchmark_result_type)
+    benchmark_type = create(:benchmark_type)
+    commit = create(:commit)
+    initiator = commit
+    benchmark_run = create(:commit_benchmark_run,
+      benchmark_result_type: benchmark_result_type,
+      benchmark_type: benchmark_type,
+      initiator: commit
+    )
+    category = "sometime"
+    BenchmarkRegressionJob.new.perform(benchmark_run, initiator, benchmark_type, benchmark_result_type, category)
+  end
+end
```

Comment:
What you're doing here is just calling the method. 

* How do you know if it is correctly detecting a regression?
* When it correctly detects a regression, what is the code expected to do?

These have to be included in the tests